### PR TITLE
Feature/operational panel UI

### PR DESCRIPTION
**Pull request by ps48**
### Description
This PR integrates UI with new backend. 

### Issues Resolved
1. Added flyout for add/replace visualization
2. Added Multi-Delete for PanelList
3. Changed PPL Placeholder and checks for index in PPL filter
4. New backend integrations
5. Removed check for time range and fields
6. Added separate css classes for panel modules

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/custom_panels/helpers/custom_input_modal.tsx [hunk 1]
```diff
@@ -64,6 +69,7 @@ export const CustomInputModal = (props: CustomInputModalProps) => {
     helpText,
     optionalArg1,
     optionalArg2,
+    optionalArg3,
```
Comment (joshuali925) [inline public/components/custom_panels/helpers/custom_input_modal.tsx:72]: might be better use `...optionalArgs` to support arbitrary number of args?

File: public/components/custom_panels/helpers/utils.tsx [hunk 2]
```diff
@@ -136,13 +166,110 @@ export const onTimeChange = (
...
+      recentlyUsedRange.start === start && recentlyUsedRange.end === end;
     return !isDuplicate;
   });
   recentlyUsedRange.unshift({ start, end });
   setStart(start);
   setEnd(end);
   setRecentlyUsedRanges(
-    recentlyUsedRange.length > 10 ? recentlyUsedRange.slice(0, 9) : recentlyUsedRange
+    recentlyUsedRange.length > 10
+      ? recentlyUsedRange.slice(0, 9)
```
Comment (joshuali925) [inline public/components/custom_panels/helpers/utils.tsx:178]: i don't think the length check is necessary? `.slice` should check it

File: server/routes/custom_panels/panels_router.ts [hunk 3]
```diff
@@ -290,11 +321,11 @@ export function PanelsRouter(router: IRouter) {
         );
         return response.ok({
           body: {
-            message: 'Panel PPL Filter Changed',
+            message: "Panel PPL Filter Changed",
           },
         });
       } catch (error) {
-        console.error('Issue in adding query filter', error);
+        console.error("Issue in adding query filter", error);
```
Comment (joshuali925) [inline server/routes/custom_panels/panels_router.ts:328]: curious what formatter are you using? not an issue but dashboards config should make them single quotes
 https://github.com/opensearch-project/OpenSearch-Dashboards/blob/ff23cc0ead19da95cb8e51cb7b2e364bda884a55/.prettierrc#L2

**Review by joshuali925 — APPROVED** [review]
minor comments

File: public/components/custom_panels/helpers/utils.tsx (hunk 2 above)
Comment (ps48) [reply public/components/custom_panels/helpers/utils.tsx:178]: makes sense! will change this.

File: public/components/custom_panels/helpers/custom_input_modal.tsx (hunk 1 above)
Comment (ps48) [reply public/components/custom_panels/helpers/custom_input_modal.tsx]: changed 👍🏽

**Pull request merged**